src/main/java/spoon/support/compiler/jdt/JDTBasedSpoonCompiler.java

```diff
@@ -279,12 +278,12 @@ public void addTemplateSources(List<SpoonResource> resources) {
 
 	@Override
 	public void setSourceOutputDirectory(File outputDirectory) {
```

Comment:
put deprecated?

Reply:
I wondered: it's still usable to put the right value in the environment...

Reply:
What do you mean?

Reply:
The method is used in some test and maybe some clients are still using it to put the value, so it can still work except now it puts the value in the Environment.

Reply:
This PR will already break the API.
I would break it more to reduce the number of API, to simplify the usage of the API.
At least I would say in the doc that this API is just a different way to access the Environment

Reply:
>I would break it more to reduce the number of API, to simplify the usage of the API.
At least I would say in the doc that this API is just a different way to access the Environment

I agree with you. I'll deprecate it then.

Reply:
I put the deprecated annotation in the interface.